airbyte-integrations/connectors/destination-jdbc/Dockerfile

```diff
@@ -8,5 +8,5 @@ COPY build/distributions/${APPLICATION}*.tar ${APPLICATION}.tar
 
 RUN tar xf ${APPLICATION}.tar --strip-components=1
 
-LABEL io.airbyte.version=0.3.5
+LABEL io.airbyte.version=0.3.6
```

Comment:
I wasn't sure if I needed to add this since the changes here are passive to this connector.